tasks/db_deploy/bin/build.sh

```diff
@@ -115,7 +115,7 @@ let return_code=$?
 check_rc $return_code "ERROR: Failed to copy lambda files to build directory."
 
 
-## Copy the lambda files to build
+## Copy the schema files to build
```

Comment:
I think this should be "## create the lambda zip files"

Reply:
Whoops. Went with removal, as it didn't actually do either of these before the next section.